name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/AbstractPDFExtractor.java

```diff
@@ -133,6 +127,16 @@ public String getFilterExtension()
             return Collections.emptyList();
         }
     }
+    
+    protected List<Item> parseDocumentTypes(List<DocumentType> documentTypes, String filename, String text) {
```

Comment:
In der Methode sehen ich erst mal keine wirklich Änderungen, oder übersehe ich etwas?